README.rst

```diff
@@ -117,9 +117,8 @@ All configuration can be done in ``setup.cfg`` like changing the description,
 URL, classifiers, installation requirements and so on as defined by setuptools_.
 That means in most cases it is not necessary to tamper with ``setup.py``.
 
-In order to build a source or wheel distribution, just run
-``tox -e build`` (``python setup.py sdist`` or ``python setup.py bdist_wheel``
-if you don't use tox_).
+In order to build a source or wheel distribution, just run ``tox -e build``
+(if you don't use tox_, you can also install ``build`` and run ``python -m build``).
```

Comment:
Really good! I was actually asked a few times if one needs ``tox`` to build, thus I think mentioning the ``build`` package here is really important. Maybe it could also be mentioned additionally in the parts of the docs where we explain how to build a package.